AdobeStockImageAdminUi/etc/adminhtml/di.xml

```diff
@@ -33,11 +33,11 @@
             </argument>
         </arguments>
     </type>
-    <type name="Magento\MediaGalleryUi\Model\SelectModifierComposite">
+    <virtualType name="Magento\MediaGalleryUi\Model\Api\SearchCriteria\CollectionProcessor\JoinProcessor" type="Magento\Framework\Api\SearchCriteria\CollectionProcessor\JoinProcessor">
```

Comment:
This virtual type is already declared in MediaGalleryUi module, so I think there is no need to specify the type
```suggestion
    <virtualType name="Magento\MediaGalleryUi\Model\Api\SearchCriteria\CollectionProcessor\JoinProcessor">
```